usr/share/rear/output/ISO/Linux-i386/250_populate_efibootimg.sh

```diff
@@ -68,7 +68,7 @@ if test "ebiso" = "$( basename $ISO_MKISOFS_BIN )" ; then
 fi
 
 if [[ -n "$(type -p grub)" ]]; then
-    cat > $efi_boot_tmp_dir/BOOTX64.conf << EOF
+    cat > $efi_boot_tmp_dir/BOOT$EFI_ARCH_UPPER.conf << EOF
```

Comment:
In this kind of situation I wouldn't mind adding braces, to make it simpler to read. But again, 80% personal preference and I would accept PRs also without

Reply:
It was my subjective choice, but not entirely arbitrary - emacs would highlight variable expansion in `BOOT$EFI_ARCH_UPPER.conf` but not in `"BOOT$EFI_ARCH_UPPER.conf"`, that's why I am adding curly braces to the latter and not the former.